### ISSUE TITLE
Distinguish admin users from non-admin users

### ISSUE THREAD

**Issue by jbphet** [phetsims/rosetta#11]
We will probably need to have some administration pages that allow us to do things like update the list of translatable simulations.  In order to do this, we will need to distinguish regular users from users with administrative privileges.  The main web site already distinguishes such users, so we should figure out how to get such information to Rosetta and incorporate it into the user validation process.

This was discussed in the 3/12/2015 design meeting, and I believe that @aaronsamuel137 said that this would require some changes to the website code.  Assigning to @aaronsamuel137 for discussion and implementation.

**Comment by aaronsamuel137** [comment]
I've updated the website code to give a "teamMember": [boolean] response in the JSON. Right now all that happens when a team member is detected is a log statement, but it seems to be working. Anything else we need to do for now on this issue?

I've also changed the url for /check-login to /services/check-login to make it more consistent with the rest of the website.

Assigning to @jbphet for review.
